quinn/src/tests.rs

```diff
@@ -129,14 +140,15 @@ fn close_endpoint() {
             }),
     );
     endpoint.close(0u32.into(), &[]);
-    runtime.run().unwrap();
+    runtime.block_on(handle).unwrap();
```

Comment:
It's important that this test, at least, require termination of both spawned tasks, as that's really what it's testing.

Reply:
Done.